src/components/itemlabal/ItemLabel.css

```diff
@@ -0,0 +1,26 @@
+.article{
```

Comment:
De classname 'article' kan misschien nog wel wat duidelijker. Als je deze class in de CSS tegenkomt, heb je geen idee wat dit zou moeten voorstellen. Dus bedenk iets dat duidelijker beschrijft wat dit is, zoals `item-label-container`